docusaurus.config.js

```diff
@@ -58,7 +58,6 @@ module.exports = {
     footer: {
       style: 'dark',
     },
-    image: 'img/og-image.png',
```

Comment:
don't think this was working, https://github.com/facebook/docusaurus/discussions/5866 instead